tests/Unit/InsertHelpdeskReplyTest.php

```
@@ -0,0 +1,296 @@
+	}
+
+	// -----------------------------------------------------------------------
+	// Happy path
+	// -----------------------------------------------------------------------
+
+	public function test_staff_reply_inserts_helpdesk_reply_comment_and_fires_action(): void {
+		$this->stub_ticket( 42 );
+		WP_Mock::userFunction( 'wp_insert_comment', array(
+			'times'  => 1,
+			'return' => 1001,
+		) );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\WP_Mock' in method 'test_staff_reply_inserts_helpdesk_reply_comment_and_fires_action'. (undefined)

(StaticAccess)